repositories/rmw_fastrtps.BUILD.bazel

```diff
@@ -0,0 +1,52 @@
+""" Builds rmw_fastrtps.
+"""
+
+load(
+    "@com_github_mvukov_rules_ros2//ros2:cc_defs.bzl",
+    "ros2_cpp_binary",
+    "ros2_cpp_library",
+)
+
+ros2_cpp_library(
+    name = "rmw_fastrtps_shared_cpp",
+    srcs = glob([
```

Comment:
Please split into hdrs and srcs.

Reply:
I think header files in `src` belong in `srcs`. They don't look like they're intended for downstream rules to #include directly.